packages/node/src/setup-config.ts

```diff
@@ -54,7 +55,7 @@ export default defineConfig({
 })
     `
 
-    await fsExtra.writeFile(join(cwd, file), content)
+    await fsExtra.writeFile(join(cwd, file), prettier.format(content))
```

Comment:
The latest version of prettier returns a promise here. So let's be sure to await that.

Reply:
since this only applies to v3+, I don't think I need to change anything currently
<img width="1057" alt="Screenshot 2023-11-03 at 13 51 16" src="https://github.com/chakra-ui/panda/assets/47224540/d7c7f17d-9ef4-4b07-a331-21666312cd9b">

will do that when we update to v3 ! 👍